catalog-info.yaml

```diff
@@ -10,4 +10,15 @@ metadata:
 spec:
   type: website
```

Comment:
What are the possible types? 🤔

Reply:
This would be for the lifecycle though, right? For type, so far, I saw `website`, `service` and `library`. Would there be other ones?

Reply:
Apologies, that's right, although you can also set set any string if that's needed by the org. so if we want to set a custom one for our own use, that's totally acceptable.

The default ones generate specific views in the catalog component so we can tailor the way we display each type:

https://github.com/cds-snc/backstage/blob/9c80931545ceb00774e3006a896f3d54d9ee1e4c/packages/app/src/components/catalog/EntityPage.tsx#L415